### ISSUE TITLE
Build broken with missing bourbon module

### ISSUE THREAD

**Issue by sukima** [newhavenio/newhavenio.github.io#80]
Just pulled down master and and followed the README and Jekyll crashes with a missing bourbon SASS module:

```
Configuration file: /home/suki/source/newhavenio.github.io/_config.yml
            Source: /home/suki/source/newhavenio.github.io
       Destination: /home/suki/source/newhavenio.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
                    File to import not found or unreadable: ../node_modules/bourbon/app/assets/stylesheets/bourbon. Load paths: /home/suki/source/newhavenio.github.io/_sass /home/suki/.rvm/gems/ruby-2.4.2@newhavenio.github.io/gems/jekyll-theme-primer-0.5.3/_sass on line 8
jekyll 3.7.3 | Error:  File to import not found or unreadable: ../node_modules/bourbon/app/assets/stylesheets/bourbon.
Load paths:
  /home/suki/source/newhavenio.github.io/_sass
  /home/suki/.rvm/gems/ruby-2.4.2@newhavenio.github.io/gems/jekyll-theme-primer-0.5.3/_sass on line 8
```

[![asciicast](https://asciinema.org/a/sqb4vZ5VC08B0jD2TlUSSOT0q.png)](https://asciinema.org/a/sqb4vZ5VC08B0jD2TlUSSOT0q)

**Comment by jnimety** [comment]
yeah, this is a known issue after the merge of the webpack PR a few days ago. As soon as someone has time to look into it we'll push a fix. Let's leave this open until that happens.

**Comment by bsutt123** [comment]
Fixed with #82
